credentials_test.go

```diff
@@ -189,6 +189,39 @@ func TestAWSGetCredentials_sharedCredentialsFile(t *testing.T) {
 	validateCredentialsProvider(credsParam, "accesskey2", "secretkey2", "", sharedConfigCredentialsSource(fileParamName), t)
 }
 
+func TestAWSGetCredentials_webIdentityToken(t *testing.T) {
```

Comment:
I've created #227 that adds tests for configuring this using environment variables or the shared config files. Those tests can be updated with support for the configuration block in addition to this test

Reply:
The PR #227 has been merged, so if you rebase this branch or merge `main` into this branch, you can pick up those tests